packages/nocodb/docker/start-litestream.sh

```diff
@@ -11,6 +11,13 @@ fi
 if [ -z "${LITESTREAM_S3_SECRET_ACCESS_KEY}" ] && [ -n "${AWS_SECRET_ACCESS_KEY}" ] ; then
   export LITESTREAM_S3_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY}"
 fi
+if [ -z "${LITESTREAM_S3_PATH}" ] && -n [ "${AWS_BUCKET_PATH}" ] ; then
```

Comment:
@pranavxc This seems to be invalid syntax, the `-n` test needs to be inside the brackets.

Reply:
Possibly I will create a follow up PR with your suggested changes.

Reply:
Could you ping me when you submit that follow-up PR? 🙂

Reply:
In between something, so once sorted out, I will work on it. I’m happy if you would like to open a PR as well.

Reply:
I've just submitted https://github.com/nocodb/nocodb/pull/8572 which should address everything we discussed here (expect for testing) :)

Reply:
Thanks for creating the PR, soon I will test it and merge.